kyo-core/js/src/main/scala/kyo/KyoAppBasePlatformSpecific.scala

```diff
@@ -0,0 +1,42 @@
+package kyo
+
+abstract class KyoAppBasePlatformSpecific extends KyoApp.Base[Throwable, Async & Resource]:
```

Comment:
Any specific reason why the error is in a separate type parameter? Could it be a single type param `Abort[Throwable] & Async & Resource`?

Reply:
Yes, I think its because of my initial attempt in which handle initially returned Fiber[E, A] < IO, so I encode both E and A on the signature. It should just encodes A and lock E to Throwable